### PR TITLE
Chore/msdk 650 polkadot version

### PR DESCRIPTION
**Pull request by monitz87**

### CODE REVIEW

File: jest.config.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 module.exports = {
-  preset: 'ts-jest',
+  // preset: 'ts-jest',
```
Comment (VictorVicente) [inline jest.config.js:2]: You should remove this comment

File: package.json [hunk 2]
```diff
@@ -42,7 +43,6 @@
     "@zerollup/ts-transform-paths": "^1.7.11",
     "case-sensitive-paths-webpack-plugin": "^2.3.0",
     "cross-env": "^7.0.0",
-    "cz-conventional-changelog": "3.3.0",
```
Comment (VictorVicente) [inline package.json:45]: Were we using this one? Do we need a replacement?

File: src/utils/__tests__/conversion.ts [hunk 3]
```diff
@@ -4217,57 +4217,57 @@ describe('transactionToTxTag', () => {
...
+//   afterAll(() => {
+//     dsMockUtils.cleanup();
+//   });
 
-  // NOTE uncomment in Governance v2 upgrade
-  // test('meshProposalStateToProposalState should convert a polkadot ProposalState object to a ProposalState', () => {
-  //   let fakeResult: ProposalState = ProposalState.Cancelled;
+//   // NOTE uncomment in Governance v2 upgrade
+//   // test('meshProposalStateToProposalState should convert a polkadot ProposalState object to a ProposalState', () => {
+//   //   let fakeResult: ProposalState = ProposalState.Cancelled;
```
Comment (VictorVicente) [inline src/utils/__tests__/conversion.ts:4235]: These and the following ones doesn't need to be commented again

**Review by VictorVicente — CHANGES_REQUESTED** [review]
You need to upgrade the GitHub Actions configuration too, to start using node v14

File: package.json (hunk 2 above)
Comment (monitz87) [reply package.json:45]: It's being used via npx

**Review by VictorVicente — APPROVED** [review]

**Pull request merged**